### PR TITLE
Fix issue choosing reply button expire

### LINKED ISSUE #323: Button of choosing reply always expires
# Steps to reproduce

- Send the following text:
    > 蔡英文的堂弟 蔡世能 ：在桃園成立大日本帝國，希望“重建日本政府”
    >
    > 若是沒有蔡英文的認同與支持，他敢這樣幹！？
- See the reply list

# Expected
User should be able to read reply when choosing reply

# Actual
"Button has been expired" error is shown

image

### PR DESCRIPTION
**Pull request by nonumpa**
Fix #323

Directly call the specific handler function when choosing article/reply for user

### CODE REVIEW

File: src/webhook/handlers/processImage.js [hunk 1]
```diff
@@ -70,10 +70,17 @@ export default async function({ data = {} }, event, userId) {
 
       // choose for user
       event = {
-        type: 'postback',
+        type: 'server_choose',
```
Comment (MrOrz) [inline src/webhook/handlers/processImage.js:73]: Question: what would happen if we just use `'postback'` here?

Comment (nonumpa) [reply src/webhook/handlers/processImage.js:73]: No functional difference.

I use `server_choose` just to sperate two different flow : "user choose" and "server choose for user".

**Comment by MrOrz** [comment]
圖片

This can be reproduced on local machine using `npm t -- --runInBand`
圖片

**Review by MrOrz — APPROVED** [review]
LGTM! Thanks for the fix 🙏

**Pull request merged**